src/ert/run_models/run_arguments.py

```diff
@@ -0,0 +1,69 @@
+from dataclasses import dataclass, field
+from typing import List, Optional
+
+
+@dataclass
+class SingleTestRunArguments:
+    simulation_mode: str = field(init=False, default="Single test run")
+
+    active_realizations: List[bool]
+    current_case: str
+    target_case: str
+    iter_num: Optional[int] = 0
+    item_number: Optional[int] = 0
+    prev_successful_realizations: Optional[int] = 0
```

Comment:
I am anal about mixing short and long forms of words. Also, the "previous" makes no sense to me? What if this were `successful_realizations`?

Reply:
Agree that it's not the best name, but this discussion is best left for another PR I think.